.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,61 +1,51 @@
 ---
 name: 🐛 Bug report
 about: Create a report to help us improve
-
 ---
+## Bug report
+<!--- Verify first that your issue is not already reported on GitHub -->
+<!--- Also test if the latest release and devel branch are affected too -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
 
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
-
-##### SUMMARY
-<!--- Explain the problem briefly -->
+### Summary
+<!--- Explain the problem briefly below -->
 
-##### ISSUE TYPE
- - Bug Report
+### Component name
+<!--- Write the name of the module, plugin, task or feature below -->
 
-##### COMPONENT NAME
-<!--- Insert, BELOW THIS COMMENT, the name of the module, plugin, task or feature.
-Do not include extra details here, e.g. "vyos_command" not "the network module vyos_command" or the full path-->
+### Ansible version
```

Comment:
+1 for not MAKING ALL HEADERS ALL-CAPS, but does Ansibullbot rely on the existing headings?

Reply:
Possibly. But let's first agree on the new format, then adapt Ansibot to accept this too.
(We will have to make sure Ansibot supports both for the foreseeable future)